Cargo.toml

```
@@ -15,10 +15,11 @@ governor = "0.6"
 tokio = { version = "1", features = ["time", "sync"] }
 once_cell = "1.18"
 thiserror = "1.0"
+serde_json = "1.0.140"
 
 [dev-dependencies]
 tokio = { version = "1", features = ["full"] }
-rama = { version = "0.2.0-alpha.6", features = ["http"] }
+rama = { version = "0.2.0-alpha.6", features = ["http", "http-full"] }
```

```suggestion
rama = { version = "0.2.0-alpha.6", features = ["http-full"] }
```

http-full already includes http, so no need to add them both, doesn't hurt, just not needed